src/components/material/MaterialHeaderText.tsx

```diff
@@ -9,11 +10,13 @@ const MaterialHeaderText: React.FC<MaterialHeaderTextProps> = ({
   title,
   author
 }) => {
+  const t = useText();
+
   return (
     <>
       <h1 className="text-header-h1 mb-16">{title}</h1>
       <p className="text-body-large">
-        <span>Af </span>
+        <span>{t("Af")} </span>
```

Comment:
The text key needs to:
* Be in english
* Be camel case
* End with "Text" suffix
* Have a name that ensures that it does not duplicate other text strings (unless it is intentional).
So in this case it could be: "MaterialHeaderAuthorByText"